### ISSUE TITLE
Training configuration for Video-finetuning

### ISSUE THREAD

**Issue by llCurious** [facebookresearch/videoseal#49]
Thx for the previous discussion on image pre-training. #33 

I have some problems on finetuning on video datasets, specifically on the parameter configuration.
```
OMP_NUM_THREADS=40 torchrun --nproc_per_node=2 train.py --local_rank 0 \
    --video_dataset myvideos --image_dataset none --workers 0 --frames_per_clip 32 \
    --resume_from /path/to/image/checkpoint.pth --resume_optimizer_state True --resume_disc True \
    --videoseal_step_size 4 --lowres_attenuation True --img_size_proc 256 --img_size_val 768 --img_size 768 \
    --extractor_model convnext_tiny --embedder_model unet_small2_yuv_quant --hidden_size_multiplier 1 --nbits 128 \
    --scaling_w_schedule None --scaling_w 0.2 --scaling_i 1.0 --attenuation jnd_1_1 \
    --epochs 601 --iter_per_epoch 100 --scheduler None --optimizer AdamW,lr=1e-5 \
    --lambda_dec 1.0 --lambda_d 0.5 --lambda_i 0.1 --perceptual_loss yuv  --num_augs 2 --augmentation_config configs/all_augs_v3.yaml --disc_in_channels 1 --disc_start 50
```
When using the above configuration, the bit accuracy drops to ~70-80%, do you have any ideas? @pierrefdz 

Perhaps:
- `--frames_per_clip 16` should we use 32, as mentioned in the paper?
- should we also set `--lambda_i 0`?
- will `--image_dataset myimages` to use hybrid training help?

Many thanks!

**Comment by pierrefdz** [comment]
Hi @llCurious, 
Sorry for the delay in answering. You can see https://github.com/facebookresearch/videoseal/issues/40 and https://github.com/facebookresearch/videoseal/issues/43#issuecomment-2995711930 for some elements of answer. 
Currently, I'd say that running the image models on video is the best thing to do. You can try finetuning the extractor only, but the increase in perf wouldnt be very noticeable.

**Comment by hadyelsahar** [comment]
Closing this, feel free to reopen if issue is not solved.
